### PR TITLE
Adding support for predicate functions and unit tests

### PR DESCRIPTION
**Pull request by davidkpiano**

### CODE REVIEW

**Comment by omnidan** [comment]
Can you document it in `README.md` too? :smiley:

**Comment by davidkpiano** [comment]
@omnidan done! Let me know if I should revise it.

File: README.md [hunk 1]
```diff
@@ -14,15 +14,26 @@ npm install --save redux-ignore
 
 ```js
 import ignoreActions from 'redux-ignore';
-ignoreActions(reducer, [ARRAY_OF_ACTIONS])
+
+// Ignore actions from an array of actions
+export default ignoreActions(reducer, [ARRAY_OF_ACTIONS])
```
Comment (omnidan) [inline README.md]: can you just list the possible options here? API should be short:
```js
import ignoreActions from 'redux-ignore';
ignoreActions(reducer, [ARRAY_OF_ACTIONS])
ignoreActions(reducer, (action) => !action.valid)
```

File: README.md [hunk 2]
```diff
@@ -44,6 +55,16 @@ combineReducers({
 Now you won't be able to increment the counter anymore, because the
 `INCREMENT_COUNTER` action is ignored.
 
+Alternatively, you can ignore actions via a predicate function:
+
+```js
+combineReducers({
+  counter: ignoreActions(counter, (action) => !action.entity === 'COUNTER')
```
Comment (omnidan) [inline README.md]: maybe use the same example as above here and filter out the type `INCREMENT_COUNTER`? or do you think it's better to use a different example?

**Comment by omnidan** [comment]
@davidkpiano I added some notes :grin:

**Comment by davidkpiano** [comment]
@omnidan Revised!

**Comment by omnidan** [comment]
@davidkpiano Thanks a lot :grin: I'll still adjust the `README.md` - hope you don't mind.

**Comment by davidkpiano** [comment]
Yeah, that's fine.

**Comment by omnidan** [comment]
@davidkpiano alright, I also added you as a collaborator and published a new npm version (1.1.0)!

**Pull request merged**